internal/internal_task_handlers.go

```diff
@@ -1441,6 +1442,20 @@ func isCommandMatchEvent(d *commandpb.Command, e *historypb.HistoryEvent, strict
 			}
 		}
 		return true
+
+	case enumspb.COMMAND_TYPE_MODIFY_WORKFLOW_PROPERTIES:
```

Comment:
@rodrigozhou @bergundy @Sushisource can you verify if I got intent of these new commands/events?